extensions/theme-defaults/themes/light_defaults.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "vscode://schemas/color-theme",
 	"name": "Light Default Colors",
 	"colors": {
-		"editor.background": "#FFFFFF",
+		"editor.background": "#FFFFFE",
```

Comment:
Did this accidentally get changed?

Reply:
I thought it's canonical:
https://github.com/microsoft/vscode/blob/196500f3647d9e26b71054652ae79e7886803117/src/vs/editor/standalone/common/themes.ts#L15
https://github.com/microsoft/vscode/blob/196500f3647d9e26b71054652ae79e7886803117/src/vs/editor/standalone/common/themes.ts#L71

Reply:
Ah, thanks for clarifying and for fixing it 😉